pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -165,9 +167,9 @@ func (r *Reconciler) reconcileService(ctx context.Context, logger *zap.SugaredLo
 			Ports: []corev1.ServicePort{{
 				Name:     eventListenerServicePortName,
 				Protocol: corev1.ProtocolTCP,
-				Port:     int32(*r.config.Port),
+				Port:     int32(port),
```

Comment:
I'm wondering -- can we get away we just changing the port on the service (and leaving the TargetPort of the EL deployment as is?). That should simplify a lot of the logic. On the other hand, it will change a few things e.g. if someone is directly port-forwarding to the EL deployment.

Thoughts @MarcelMue @savitaashture @bigkevmcd ?

Reply:
I agree that we could simplify this logic but can not tell what the intended setup is. Do we know of any users which would be impacted by such a change?

Reply:
I can change it but would prefer to do it in a follow up, if that's okay. I don't want to mix all these changes together :)